### ISSUE TITLE
orbi_plot_shot_noise() isotopocule filter

### ISSUE THREAD

**Issue by brettdavidheiser** [isoverse/isoorbi#44]
If you have lot of isotopocules the shot noise plots get very busy, especially when facet_wrap. It might be nice to have a built in filter like as in the fuction call `orbi_plot_raw_data(isotopocules = c("13C"), y= ratio)`. Of course it can be done with a filter call before hand.
`flageddata_w_ratios |> orbi_analyze_shot_noise() |> dplyr::filter(isotopocule %in% c("13C", "18O")) |> orbi_plot_shot_noise()`

**Comment by SabrielxD** [comment]
or it can be done with `orbi_filter_isox()`, no?

**Comment by brettdavidheiser** [comment]
Yes you can use the `orbi_filter_isox()`. It looks like you also have to filter out the data that has been labeled "unused" in the data_type before calling the shot noise function

**Comment by sebkopf** [comment]
I'm in favor of providing an isotopocules parameter since we have that in the other plotting functions it would be consistent (default would be to include all). Could also include a flag whether to show unused data but that's something we could consider for all plotting functions. Default should probably be `show_unused = TRUE`, yes?

**Comment by sebkopf** [comment]
moving towrads using `orbi_filter_isotopocules()`
